projects/RPi/devices/RPi4/patches/bluez/bluez.patch

```diff
@@ -1,58 +0,0 @@
-The 32-bit SIOCGSTAMP has been deprecated. Use the deprecated name
```

Comment:
Should also be removed:
projects/Generic/patches/bluez/bluez.patch
projects/NXP/devices/iMX6/patches/bluez/bluez-01-tools-fix-siocgstamp.patch
projects/OdroidXU3/patches/bluez/bluez-32bit.patch

Reply:
Removed all three, thx